### PR TITLE
Multiclass Neural Net Classifier

### PR DESCRIPTION
**Pull request by WildCayote**
Created a tensorflow model for classifying the games as home , draw or win. Then for the home_double and away_double I wrote a simple logic in the ml_package/predict.py to obtain those probabilities by adding home with draw and away with draw respectively. 

Updated requirements.txt to look for tensorflow versions greater than or the same as 2.10.1 and not just that specific version.

### CODE REVIEW

**Comment by WildCayote** [comment]
| Metric                                 | main    | workspace   | Change   |
|----------------------------------------|---------|-------------|----------|
| away_double.metrics.accuracy_score     | 0.63105 | -           | -        |
| away_double.model_type                 | GDBoost | -           | -        |
| away_win.metrics.accuracy_score        | 0.71463 | -           | -        |
| away_win.model_type                    | GDBoost | -           | -        |
| draw.metrics.accuracy_score            | 0.77583 | -           | -        |
| draw.model_type                        | votting | -           | -        |
| home_double.metrics.accuracy_score     | 0.71463 | -           | -        |
| home_double.model_type                 | GDBoost | -           | -        |
| home_win.metrics.accuracy_score        | 0.63105 | -           | -        |
| home_win.model_type                    | GDBoost | -           | -        |
| multiclass.metrics.accuracy            | -       | 0.52595     | -        |
| multiclass.metrics.accuracy_score      | -       | 0.52831     | -        |
| multiclass.metrics.loss                | -       | 0.98715     | -        |
| multiclass.metrics.val_accuracy        | -       | 0.52705     | -        |
| multiclass.metrics.val_loss            | -       | 0.9848      | -        |
| multiclass.metrics.validation_accuracy | -       | 0.52705     | -        |
| multiclass.metrics.validation_loss     | -       | 0.9848      | -        |
| multiclass.model_type                  | -       | Nural Net   | -        |
